Mathlib/Analysis/RCLike/Lemmas.lean

```diff
@@ -1,15 +1,53 @@
 /-
 Copyright (c) 2020 Frédéric Dupuis. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Frédéric Dupuis
+Authors: Frédéric Dupuis, Yaël Dillies
 -/
+import Mathlib.Algebra.Group.AddChar
 import Mathlib.Analysis.Normed.Module.FiniteDimension
 import Mathlib.Analysis.RCLike.Basic
 
 /-! # Further lemmas about `RCLike` -/
 
+open Function
+open scoped ComplexConjugate
 
-variable {K E : Type*} [RCLike K]
+variable {G K E : Type*}
+
+namespace AddChar
+section AddGroup
+variable [AddGroup G]
+
+section NormedField
+variable [Finite G] [NormedField K]
```

Comment:
This lemma and the one below it don't look like they belong in this file - they don't mention RCLike at all?

Reply:
Maybe, but I don't know where else to put that one lemma

Reply:
(marking as resolved since the lemma has now been moved elsewhere)